src/autopas/molecularDynamics/LJFunctorCuda.cu

```diff
@@ -786,9 +886,15 @@ __global__ void CellVerletTraversalN3(LJFunctorCudaSoA<floatType> cell, unsigned
 
     cell2_pos_shared[threadIdx.x] = {cell._posX[cellStart + threadIdx.x], cell._posY[cellStart + threadIdx.x],
                                      cell._posZ[cellStart + threadIdx.x]};
+    cell2_ownershipState_shared[threadIdx.x] = cell._ownershipState[cellStart + threadIdx.x];
     __syncthreads();
-    for (int j = 0; j < block_size; ++j) {
-      myf = bodyBodyF<floatType>(myposition, cell2_pos_shared[j], myf);
+
+    if (myOwnershipState != OwnershipState::dummy) {
+      for (int j = 0; j < block_size; ++j) {
+        if (cell2_ownershipState_shared[j] != OwnershipState::dummy) {
+          myf = bodyBodyF<floatType>(myposition, cell2_pos_shared[j], myf);
+        }
+      }
```

Comment:
so... much... code... duplication! Feels like I just read that block 1000 times. But I guess refactoring this would mean rewriting basically the whole functor?

Reply:
yup, as we are planning to delete it anyway, I wouldn't recommend doing that...